packages/web-components/fast-components/src/slider/slider.ts

```diff
@@ -254,9 +254,13 @@ export class Slider extends FormAssociated<HTMLInputElement>
         if (this.readOnly || this.disabled || e.defaultPrevented) {
             return;
         }
+
         // update the value based on current position
         const eventValue: number =
-            this.orientation === Orientation.horizontal ? e.pageX : e.pageY;
+            this.orientation === Orientation.horizontal
+                ? e.pageX - this.getBoundingClientRect().left
```

Comment:
I'm a bit concerned about calling `getBoundingClientRect` in a mouseMove event handler since it can fire very rapidly - `getBoundingClientRect` is expensive and forces a layout which can be janky - how does this feel? Is there a way to accomplish this *without* `getBoundingClientRect`?

Reply:
We could reduce the number of calls by caching the value during the `mousedown` for use during the `mousemove`, or by implementing a ResizeObserver (which isn't supported in Firefox Android).

There are other performance concerns I noticed while poking around in this component, and it would probably be good to come back around to optimize things (particularly the `transition: all` part).

Reply:
If you think an all-up perf pass on it is necessary lets take that as a follow-up and address this there - I don't want to block progress but could see this being an issue.

Reply:
@nicholasrice https://github.com/microsoft/fast-dna/issues/3126